backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dialog/BpkDialog.kt

```diff
@@ -30,6 +30,7 @@ import net.skyscanner.backpack.compose.dialog.internal.Dialog
 import net.skyscanner.backpack.compose.icon.BpkIcon
 
 @Composable
+@Suppress("ModifierMissing")
```

Comment:
didn't we want to add the modifier as it also causes the lint warning on the app side when using the dialog and having a composable for it?

Reply:
Yes! But I think it should be in the next ticket, this is an API change.